lib/workers/repository/config-migration/branch/rebase.ts

```diff
@@ -22,9 +26,9 @@ export async function rebaseMigrationBranch(
     return null;
   }
   const configFileName = migratedConfigData.filename;
-  const contents = migratedConfigData.content;
+  let contents = migratedConfigData.content;
   const existingContents = await getFile(configFileName, branchName);
-  if (contents === existingContents) {
+  if (hash(contents) === hash(existingContents)) {
```

Comment:
what's the benefit of additional hashing? i think this would just slow down and should avoided here